wire-ios-data-model/Source/Model/FeatureConfig/AppLock/AppLockController.swift

```diff
@@ -116,7 +116,11 @@ public final class AppLockController: AppLockType {
 
     // MARK: - Life cycle
 
-    public init(userId: UUID, selfUser: ZMUser, legacyConfig: LegacyConfig? = nil) {
+    public init(
+        userId: UUID,
+        selfUser: ZMUser,
+        legacyConfig: LegacyConfig? = nil
```

Comment:
```suggestion
        legacyConfig: LegacyConfig?
```
In general there is a chance one forgets to set arguments if they have default values. In this case the default value can be removed, because no caller uses it :)